io/disk/lvsetup.py

```diff
@@ -45,11 +46,20 @@ def setUp(self):
         """
         Check existence of input PV,VG, LV and snapshots prior to Test.
         """
+        pkgs = []
+        smm = SoftwareManager()
         self.disk = self.params.get('disk', default=None)
         vg_name = self.params.get('vg_name', default='avocado_vg')
         lv_name = self.params.get('lv_name', default='avocado_lv')
         self.lv_size = self.params.get('lv_size', default='1G')
         self.fs_name = self.params.get('fs', default='ext4')
+        if self.fs_name == 'xfs':
+            pkgs.append('xfsprogs')
+        if self.fs_name == 'btrfs':
+            pkgs.append('btrfs-progs')
+        for pkg in pkgs:
```

Comment:
i don't think it can be requirement if some body want to add more package (if you want to add more packages please add in this PR ) that time it will implement in this way as this not looks good